docs/abi-spec.rst

```diff
@@ -485,12 +486,13 @@ For example,
 
     pragma solidity >=0.5.0 <0.7.0;
 
+
     contract Test {
-      constructor() public { b = hex"12345678901234567890123456789012"; }
-      event Event(uint indexed a, bytes32 b);
-      event Event2(uint indexed a, bytes32 b);
-      function foo(uint a) public { emit Event(a, b); }
-      bytes32 b;
+        constructor() public { b = hex"12345678901234567890123456789012"; }
+        event Event(uint indexed a, bytes32 b);
+        event Event2(uint indexed a, bytes32 b);
+        function foo(uint a) public { emit Event(a, b); }
+        bytes32 b;
```

Comment:
I'm not sure you want to do that in this PR, but according to https://solidity.readthedocs.io/en/v0.5.8/style-guide.html#order-of-layout this line should come first, followed by the events and then the functions.

Reply:
True… That may be a hard one to check and I doubt solhint supports it, but good call, I may raise that as an issue anyway.